### ISSUE TITLE
Use of jQuery

### ISSUE THREAD

**Issue by The-Penultimate-Defenestrator** [Semantic-Org/Semantic-UI#3979]
I'd like to discuss the possibility of dropping jQuery more thoroughly. #1175 is closed, which sort of indicates that discussion is closed. I think it should be reopened.

How much of this project uses jQuery? How much of that could be accomplished with modern JavaScript? If the transitions and animations are done in jQuery, how much of that could be rewritten in CSS3?

From ["You Might Not Need jQuery"](http://youmightnotneedjquery.com):

> If you're developing a library on the other hand, please take a moment to consider if you actually need jQuery as a dependency. Maybe you can include a few lines of utility code, and forgo the requirement.

Maybe two versions of this library could be released, one using jQuery, and another focused on newer browsers, using vanilla JavaScript. The aforementioned page includes several ways in which jQuery support can be replaced with a cross-browser vanilla JS solution.

So, in short, how feasible *is* a jQuery removal? Is this planned, or at least open to further discussion, even though #1175 is closed?

**Comment by avalanche1** [comment]
Reopened https://github.com/Semantic-Org/Semantic-UI/issues/1175
